packages/gatsby-source-filesystem/src/gatsby-node.js

```diff
@@ -96,23 +131,17 @@ const createFSMachine = (
     },
     {
       actions: {
-        createAndProcessNode(_, { pathType, path }, { state }) {
-          if (state.matches(`BOOTSTRAP.BOOTSTRAPPED`)) {
-            reporter.info(`added ${pathType} at ${path}`)
-          }
+        createAndProcessNode(_, { pathType, path }) {
           createAndProcessNode(path).catch(err => reporter.error(err))
         },
         deletePathNode(_, { pathType, path }, { state }) {
-          if (state.matches(`BOOTSTRAP.BOOTSTRAPPED`)) {
-            reporter.info(`${pathType} deleted at ${path}`)
-          }
           deletePathNode(path)
         },
         flushPathQueue(_, { resolve, reject }) {
           flushPathQueue().then(resolve, reject)
         },
         queueNodeDeleting(_, { path }) {
-          pathQueue.push({ op: `delete`, path })
+          pathQueue.push({ op: `delete `, path })
```

Comment:
Something weird happened to formatting in latest commit - also check lines 159-166 and L186 (using new line numbers)
```suggestion
          pathQueue.push({ op: `delete`, path })
```

Reply:
oh, almost outdated ;)

Reply:
yeah I was checking my changes and saw something weird